operator/templates/mirror-maker.yaml

```diff
@@ -18,7 +18,7 @@ spec:
       containers:
       - name: kafka-mirror-maker
         image: mesosphere/kafka:2.5.1-1.3.1
-        imagePullPolicy: Always
+        imagePullPolicy: {{ .Params.KAFKA_IMAGE_PULL_POLICY }}
```

Comment:
should we have a separate policy here or is this ok?

Reply:
See my comment above.